docs/examples/mnist-convnet.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Convolutional networks\n",
+    "# MNIST convolutional network\n",
```

Comment:
Should we add a note to this notebook (in the preamble) just saying that it doesn't use block splitting, and to take a look at the CIFAR-10 notebook for an example of that?

Reply:
I added it a little farther down in the notebook, as there seemed to be a natural spot to talk about it when we discuss the weird parallel architecture we use in the MNIST convnet.